app/bundles/UserBundle/Security/Provider/UserProvider.php

```diff
@@ -37,6 +37,11 @@ public function __construct(
      * @return User
      */
     public function loadUserByUsername($username)
```

Comment:
Shall we deprecate the methods that are now just proxies to the new methods?

Reply:
They are deprecated in the interface already.